src/v_video.c

```diff
@@ -592,11 +599,49 @@ void V_DrawRawScreen(byte *raw)
 }
 
 //
+// V_InitScreenShotScaler
+// find a scaler for writing out PNG screenshots
+//
+
+static void V_InitScreenShotScaler()
+{
+    screen_mode_t * mode;
+    byte *doompal;
+
+    mode = I_FindScreenMode(screenshot_width, screenshot_height, 0);
+
+    if (!mode)
+    {
+        return;
+    }
+
+    if (mode->InitMode)
+    {
+        doompal = W_CacheLumpName(DEH_String("PLAYPAL"), PU_CACHE);
+
+        if (!doompal)
+        {
+            return;
+        }
+
+        mode->InitMode(doompal);
+    }
+
+    screenshot_width = mode->width;
+    screenshot_height = mode->height;
+    screenshot_scaler = mode;
+}
+
+//
 // V_Init
-// 
+//
 void V_Init (void) 
 { 
-    // no-op!
+    // sensible default
+    screenshot_width = SCREENWIDTH * 5;
```

Comment:
Probably not, you're right. I refactored V_InitShotSomething since putting them there, I think it made more sense in an earlier version :)